layout: default
# Yiming Hao

Email: [y7hao@ucsd.edu](mailto:y7hao@ucsd.edu)

## Section Information

- **Course Title**: Upscaling from Old, Boring Microbiome Data to New Data That Will Change the World
- **Section**: A06
- **Mentor**: Rob Knight

## Reflections and Plans for Quarter 2 Project

### What is the most interesting topic covered in your domain this quarter?
The most intriguing topic this quarter has been the identification of antimicrobial peptides (AMPs) from the human gut microbiome using deep learning techniques. The blend of computational biology with advanced machine learning models like LSTM, BERT, and Attention mechanisms presents a novel approach to tackling antibiotic resistance.

### Describe a potential investigation you would like to pursue for your Quarter 2 Project.
For the Quarter 2 Project, I am interested in expanding our current AMP identification model to incorporate network analysis. This would enable us to efficiently narrow down potential AMP candidates, focusing on unique peptides that differ significantly from existing antibiotics, thereby accelerating the discovery of new antimicrobial agents.

### What is a potential change youâ€™d make to the approach taken in your current Quarter 1 Project?
A potential change in our Quarter 1 Project could be the integration of a more diverse set of data sources. By including more varied datasets, such as metagenome and metaproteomic data, we could potentially enhance the model's accuracy and robustness in identifying AMPs across a broader spectrum of microbial environments.

### What other techniques would you be interested in using in your project?
I am particularly interested in exploring the use of convolutional neural networks (CNNs) and network analysis in conjunction with our existing deep learning models. CNNs could provide a more nuanced understanding of peptide sequences, while network analysis could offer a strategic approach to selecting the most promising AMP candidates for further experimental validation.
